src/handlers/check_commits/no_mentions.rs

```diff
@@ -4,12 +4,24 @@
 use crate::{config::NoMentionsConfig, github::GithubCommit};
 
 pub(super) fn mentions_in_commits(
-    _conf: &NoMentionsConfig,
+    pr_title: &str,
+    conf: &NoMentionsConfig,
     commits: &[GithubCommit],
 ) -> Option<String> {
+    if conf
+        .exclude_titles
+        .iter()
+        .any(|s| pr_title.to_lowercase().contains(s))
```

Comment:
I think `s` (the excluded substring) also needs to be lowercased, otherwise an uppercased excluded title won't ever match with the always lowercased PR title.